main.go

```diff
@@ -73,14 +73,12 @@ func main() {
 	var enableLeaderElection bool
 	var probeAddr string
 	var odfConsolePort int
-	var ibmConsolePort int
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8085", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8082", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	flag.IntVar(&odfConsolePort, "odf-console-port", 9001, "The port where the ODF console server will be serving it's payload")
```

Comment:
I would suggest you to keep the line as it was before and just delete the IBM related ones.

Reply:
ack. one commit only removing, other commits can rename / cleanup etc.